### DISCUSSION TITLE
I want to stay accountable to complete this pytorch-deep-learning - Day1

### DISCUSSION THREAD

**Discussion by jannat226** [mrdbourke/pytorch-deep-learning#1195] [General]
Today i learnt about tensors, attributes of tensor, tensor multiplication, restacking,squeeze, unsqueeze, stacking all these and numpy in pytorch - thank you so much for such good resources

**Comment by jannat226** [comment]
Day 2 -> Today i learnt about how the model actually works , loss function , optimizer , training loop , testing loop how to save my model and how to load it.Thank you so much for such good resources and I loved the song that you sung for the training and testing loop.

**Comment by mujtabashahbaz** [comment]
Hey @jannat226! Fellow learner here... Your progress is inspiring. The training loop song made those concepts stick so well :) Let’s keep motivating each other... We’ve got this!
